modules/userdirectory-brightspace/src/main/java/org/opencastproject/userdirectory/brightspace/client/api/BrightspaceUser.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Licensed to The Apereo Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ *
+ * The Apereo Foundation licenses this file to you under the Educational
+ * Community License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License
+ * at:
+ *
+ *   http://opensource.org/licenses/ecl2.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+
+package org.opencastproject.userdirectory.brightspace.client.api;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class BrightspaceUser {
+
+  private String orgId;
+  private String userId;
+  private String firstName;
+  private String middleName;
+  private String lastName;
+  private String userName;
+  private String externalEmail;
+  private String orgDefinedId;
+  private String uniqueIdentifier;
+  private Activation activation;
+  private String displayName;
+
+  @JsonCreator
```

Comment:
We're using a different library for these kinds of annotations across the rest of the codebase.  Any chance this can be adapted to use that library?  I can see this kind of module not getting security updates since it's using a different library that we might not be checking as closely.

Reply:
I can have a look, could you point me in the right direction to where to find the different library ?

Reply:
Sure thing.  Most of our classes with these annotations are prefixed with "Jaxb".  A simple one would be https://github.com/opencast/opencast/blob/develop/modules/common/src/main/java/org/opencastproject/job/api/JaxbJob.java, and its other Jaxb friends in that package.

Reply:
I had a look to change this myself but not really confident enough that I won't break more than it'll fix. We'll have a look to see if we can get this fixed by a developer.